src/FSharp.Stats/Fitting/GeneralisedLinearModel.fs

```diff
@@ -0,0 +1,697 @@
+namespace FSharp.Stats.Fitting.GLM
+
+
+open System
+open FSharp.Stats
+open Algebra.LinearAlgebra 
+
+/// <summary>
+/// Represents the distribution families for Generalized Linear Models (GLMs).
+/// </summary>
+type GlmDistributionFamily =
+    /// Normal distribution family.
+    | Normal
+    /// Exponential distribution family.
+    | Exponential
+    /// Gamma distribution family.
+    | Gamma
+    /// Inverse Gaussian distribution family.
+    | InverseGaussian
+    /// Poisson distribution family.
+    | Poisson
+    /// Bernoulli distribution family.
+    | Bernouli
+    /// Binomial distribution family.
+    | Binomial
+    /// Categorical distribution family.
+    | Categorical
+    /// Multinomial distribution family.
+    | Multinomial
+
+/// <summary>
+/// Represents a collection of link functions used in a generalized linear model.
+/// </summary>
+type LinkFunctions =
+    | GetLink of (float -> float)
+    | GetInvLink of (float -> float)
+    | GetInvLinkDerivative of (float -> float)
+
+/// <summary>
+/// Represents a link function used in a generalized linear model.
+/// </summary>
+type LinkFunction =
+    {
+        /// Gets the link function.
+        getLink: float -> float
+        /// Gets the inverse link function.
+        getInvLink: float -> float
+        /// Gets the derivative of the link function.
+        getDeriv: float -> float
+        /// Gets the derivative of the inverse link function.
+        getInvLinkDerivative: float -> float
+    }
+
+/// <summary>
+/// Represents the return type of a Generalised Linear Model (GLM).
+/// </summary>
+/// <remarks>
+/// This type contains the following elements:
+/// <list type="bullet">
```

Comment:
I'm not sure if this is supported by [XML-Documentation](https://learn.microsoft.com/en-us/dotnet/fsharp/language-reference/xml-documentation#recommended-tags). In VSCommunity, the styling is not applied. 

I checked if the API-Reference was displayed correctly using `dotnet fsdocs watch --eval` and it didn't work. 

Does it work on your machine? If not, please use the allowed tags only